README.md

```diff
@@ -9,3 +9,6 @@ This is a game being developed for the 1M Computer Game Development II class.
 
 ## Mission
 Build synergistic relationships which empower people to help each other and serve a wider community.
+
+### Game Overview
+The game, in the current ideation, is merely an environment. Hey, but that's ok.
```

Comment:
I think thats a great overview of the game, and the sightly humorous tone shows our idea well.